Makefile

```diff
@@ -19,8 +15,7 @@ STANDALONE := extract-types  generate-buildhook yaml2json-procfile
 # Docker Root FS
 BINDIR := ./rootfs
 
-# Legacy support for DEV_REGISTRY, plus new support for DEIS_REGISTRY.
-DEV_REGISTRY ?= $$DEV_REGISTRY
+DEV_REGISTRY ?= $(shell docker-machine ip deis):5000
```

Comment:
we recently removed this from etcd to get the deployments working again: https://github.com/deis/etcd/pull/19

travis was failing because docker-machine was not found, even though we explicitly added DEIS_REGISTRY.